R/article.R

```diff
@@ -176,6 +176,18 @@ frontiers_article <- function(..., keep_tex = TRUE) {
   pdf_document_format("frontiers", keep_tex = keep_tex, ...)
 }
 
+
+#' @section \code{glossa_article}: Format for creating submissions to
+#'   Glossa: a journal of general linguistics. Adapted from
+#'   \url{https://github.com/guidovw/Glossalatex}.
+#' @export
+#' @rdname article
+glossa_article <- function(..., keep_tex = TRUE, latex_engine = "xelatex") {
+  pdf_document_format(
+    "glossa", keep_tex = keep_tex, latex_engine = latex_engine, ...
+  )
```

Comment:
`citation_package = "default"` will be used here by default. It means Pandoc citeproc will be used, not `natbib` or `biblatex`. Did you test it ? Is this ok ? 

There is no bibliography file to test it so it is currently not used in the doc.